<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Channels</class>
 <widget class="QWidget" name="Channels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="primary">
    <widget class="QListWidget" name="listWidget_Channels">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>951</width>
       <height>461</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_newChannel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new channel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="secondary">
    <widget class="QPushButton" name="pushButton_back">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_channel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>49</y>
       <width>971</width>
       <height>511</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="posts">
      <widget class="QPushButton" name="pushButton_send">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>470</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_textMaker">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>470</y>
         <width>671</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_CreateMeet">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Create a meeting</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_Posts">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>1</y>
         <width>961</width>
         <height>461</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="files">
      <widget class="QListWidget" name="listWidget_Files">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>951</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_UploadFile">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>470</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_DownloadFile">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>470</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="details">
      <widget class="QTextBrowser" name="textBrowser_Details">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>971</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_Posts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Posts</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Files">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Files</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Details">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_channelName">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Channel Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AddMember">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add member</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
